content/product-engineering/engineering/hiring/index.md

```diff
@@ -15,3 +15,20 @@ Most of our process is documented in a private repository, but our goal is to mo
 ## Open positions
 
 See our [careers page](https://boards.greenhouse.io/sourcegraph91) for open positions and more info.
+
+## Interview polices for Sourcegraph teammates
+
+### Frontend positions
+
+Since there is a smaller number of Sourcegraph teammates with frontend expertise (as compared to backend), we have a different policy for distributing tasks related to hiring. Our policies on frontend interviews and grading are:
+
+1. All frontend engineers are expected to (eventually) participate in hiring, as long as they have been in their current position for at least three months.
+1. To gain experience with a particular type of interview/assignment, the engineer should pair at least once with someone who is experienced.
+1. For _interviews_, an interviewer from the same team or org as the open position is strongly preferred. If this is not possible (e.g., due to PTO, timezone availability, or interviewer load), any other frontend engineer with that type of interview may be selected.
+1. For _assignment grading_, we aim to distribute the load as evenly as possible across all frontend engineers; there is no need to scope the assignments to a particular team/org.
```

Comment:
the feedback I received from my team - it would be great if the same person does coding task assessment and follow-up interview so that this person has all the context.

Reply:
Are you referring to the [follow-up after a frontend coding exercise](https://handbook.sourcegraph.com/product-engineering/engineering/hiring/software-engineer-coding-exercise#frontend-coding-exercise)?

> If we decide to move forward, we schedule a one hour follow-up call to discuss your submission.

If so, I agree. I think that's the implicit assumption but it's worth making it implicit.

However, for other interviews where the interview doesn't have anything to do with the previous assignment, my feeling is that it's actually better have a different set of people grading than those interviewing, for a couple reasons:

1. Better load distribution: ideally you want the people on the team to do the in-person interview, but if they do _all_ of the grading too, it's a lot of work.
2. More diverse signal. Having different people do the grading and interviewing gives you more signal in the hiring process.

Reply:
I've added a note about the follow-up calls.